fpm-entrypoint.sh

```diff
@@ -18,6 +18,7 @@ elif [ "$RESTY_IMAGE_BASE" == "centos" ] || [ "$RESTY_IMAGE_BASE" == "rhel" ] ||
   fi
   if [ "$RESTY_IMAGE_BASE" == "amazonlinux" ]; then
     OUTPUT_FILE_SUFFIX=".aws"
+    FPM_PARAMS="$FPM_PARAMS -d /usr/sbin/useradd"
```

Comment:
this should move to lines 10 and 14 I think right (or somewhere after line 32 would work)?

We want to make `/usr/sbin/useradd` a dependency for everything but alpine correct?

Reply:
`useradd` is not available by default on `amazonlinux` so that should be the only place where we need to install it, the other distros have it.